docs/components/modeler/web-modeler/git-sync.md

```diff
@@ -0,0 +1,102 @@
+---
+id: git-sync
+title: Git sync
+description: Connect Web Modeler to your Git repositories to keep your projects synced.
+---
+
+Organization owners and administrators can connect their Web Modeler projects to GitHub, allowing users to keep their Web Modeler, Desktop Modeler, and official version control projects synced.
+
+Once the basic integration is configured by an organization owner or organization administrator, project administrators and editors can use the built-in button to pull changes from GitHub, integrate contributions from Desktop Modeler users, and merge their own work.
+
+## Connect to GitHub
+
+### Create a new GitHub App
+
+Web Modeler requires a GitHub App to sync changes with your GitHub repository.
+
+Follow the [GitHub documentation](https://docs.github.com/en/apps/creating-github-apps/registering-a-github-app/registering-a-github-app) to create a new GitHub App for your organization or account with the following configuration:
+
+- Under **Webhooks**, deselect **Active**
+- Under **Permissions > Repository permissions**, enable **Read and write** for the following options:
+  - Commit statuses
+  - Contents
+  - Pull requests
+
+Click **Create GitHub App** to finish.
+
+### Generate a private key
+
+1. In your new application's setting page, navigate to **General > Private keys**.
+2. Select **Generate a private key**. This key is needed for Web Modeler to access your repository, and is automatically downloaded when created.
+3. Save the key for use in Web Modeler.
+
+### Install the GitHub App
+
+1. In your application's setting page, navigate to **Install app**.
+2. Click on the **Install** button for your organization or account.
+3. Select **Only select repositories**, and choose the repository to sync with Web Modeler.
+4. Once redirected to your application's installation page, copy the **Installation ID** located at the end of the page's URL: `https://github.com/settings/installations/{installation_id}`.
+
+### Configure GitHub in Web Modeler
+
+:::note
+
+An organization administration account (or project administrator in Camunda Self-Managed) is required for the initial GitHub configuration.
+
+:::
+
+1. Within Web Modeler, navigate to the process application you would like to connect to GitHub, and click **Connect GitHub**.
+
+2. Provide the following information in the GitHub Configuration modal:
+
+   - **Installation ID:** Found in your GitHub App's settings page.
+   - **Client ID:** Found in your GitHub App's settings page.
+   - **Private Key:** Found in your GitHub App's settings page.
```

Comment:
💭 also here, we should probably mention that the downloaded PEM file can be opened with a text editor, and its content can be copy/pasted